presto-cassandra/src/main/java/io/prestosql/plugin/cassandra/CassandraType.java

```diff
@@ -299,7 +299,7 @@ public String getColumnValueForCql(Row row, int position)
             case TIMESTAMP:
                 return Long.toString(row.getTimestamp(position).getTime());
             case DATE:
-                return row.getDate(position).toString();
+                return quoteStringLiteral(row.getDate(position).toString());
```

Comment:
Seems this fixes something, not adds support for, would it be possible to have a test for this?

Also, perhaps this should be a separate commit

Reply:
This is fixing the partition key push down use case for date type and it is needed to resolve this issue